src/Mgate/PersonneBundle/Resources/translations/person.fr.yml

```diff
@@ -0,0 +1,155 @@
+generic:
```

Comment:
As stated in the readme, this is for all Junior Entreprises, and I think that other developpers will join easier this way (+ english is a must for devs)